messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/MessagingSubsystemParser_1_0.java

```diff
@@ -457,8 +457,8 @@
                                                         ConnectionFactoryAttributes.Common.DUPS_OK_BATCH_SIZE,
                                                         ConnectionFactoryAttributes.Common.TRANSACTION_BATCH_SIZE,
                                                         ConnectionFactoryAttributes.Common.BLOCK_ON_ACKNOWLEDGE,
-                                                        ConnectionFactoryAttributes.Common.BLOCK_ON_DURABLE_SEND,
                                                         ConnectionFactoryAttributes.Common.BLOCK_ON_NON_DURABLE_SEND,
+                                                        ConnectionFactoryAttributes.Common.BLOCK_ON_DURABLE_SEND,
```

Comment:
@jmesnil different parts of the xsd (legacy-connection-factory vs the attribute group used for the other factory elements. The attribute group one has non-durable first:

https://github.com/wildfly/wildfly/blob/master/messaging-activemq/src/main/resources/schema/wildfly-messaging-activemq_1_0.xsd#L704

This reminded me though that I meant to check legacy-connection-factory and forgot. So I just did and added another commit for that one.